benchmark/Math/NumberTheory/GaussianIntegersBench.hs

```diff
@@ -21,6 +21,6 @@ benchTau n = bench (show n) $ nf (\m -> sum [tau (x :+ y) | x <- [1..m], y <- [0
 
 benchSuite :: Benchmark
 benchSuite = bgroup "Gaussian"
-  [ bgroup "findPrime" $ map benchFindPrime [1000033, 10000121, 100000037, 1000000009, 10000000033, 100000000057, 1000000000061, 10000000000037]
+  [ bgroup "findPrime" $ map benchFindPrime [1000033, 10000121, 100000037, 1000000021, 10000000033, 100000000057, 1000000000061, 10000000000037]
```

Comment:
What was the reason for change?

Reply:
Typo. I was using this file by mistake to write newer primes with the same order of magnitude but of the form `3k + 1` which Eisenstein integers require, instead of `4k + 1`, which is what Gaussian integers require.